pytorch_lightning/loggers/tensorboard.py

```diff
@@ -286,6 +287,12 @@ def _get_next_version(self):
 
         return max(existing_versions) + 1
 
+    @staticmethod
+    def _sanitize_params(params: Dict[str, Any]) -> Dict[str, Any]:
+        params = LightningLoggerBase._sanitize_params(params)
+        # logging of arrays with dimension > 1 is not supported, sanitize as string
+        return {k: str(v) for k, v in params.items() if isinstance(v, (torch.Tensor, np.ndarray)) and v.ndim > 1}
```

Comment:
```suggestion
        return {k: str(v) if isinstance(v, (torch.Tensor, np.ndarray)) and v.ndim > 1 else v for k, v in params.items() }
```

We should keep all params that are no tensors or arrays as well I think :)

Reply:
absolutely!